src/affine.rs

```diff
@@ -91,6 +91,21 @@ impl Affine {
             inv_det * (self.0[1] * self.0[4] - self.0[0] * self.0[5]),
         ])
     }
+
+    /// Compute the bounding box of a transformed rectangle.
+    ///
+    /// Returns the minimal `Rect` that encloses given `Rect` after affine transformation.
```

Comment:
maybe _the_ given `Rect`? I'm actually not sure if this is a typo or a style choice.